oxidation/libparsec/crates/platform_local_db/src/executor.rs

```diff
@@ -60,11 +72,21 @@ impl SqliteExecutor {
 
 impl Drop for SqliteExecutor {
     fn drop(&mut self) {
-        if let Err(e) = self.job_sender.send(ExecOrStop::Stop) {
-            eprintln!("Cannot send message to the background executor (Could just mean the executor is closed): {e}")
+        if self.job_sender.send(ExecOrStop::Stop).is_err() {
+            // If the send fails, this is most likely because the executor
+            // thread is already stopped. This might be because:
+            // - the thread has panicked
+            // - a bug in the code has cause another `ExecOrStop::Stop` to be
+            //   send to the thead before
```

Comment:
so you agree if `ExecOrStop::Stop` is send twice it's a bug in our code ? or there is something else ? 🤔

Reply:
If `Stop` is sent twice it's a flaw, meaning we would have `drop(executor)` multiple and `rust` won't allow to compile a code like that

Reply:
> If Stop is sent twice it's a flaw

Yes, that is my point saying in the comment this case is a bug

> meaning we would have drop(executor) multiple

You are making assumption on how bug could work.
This could also be that for whatever reason `ExecOrStop::Stop` is send where `ExecOrStop::Exec` was supposed to be send.